### PR TITLE
Streamlining samples

### PR DESCRIPTION
**Pull request by morgantaschuk**
Reducing the number of samples/libraries to add. Fixing some suggestions from the training.

### CODE REVIEW

File: 4-samples.md [hunk 1]
```diff
@@ -15,31 +15,86 @@ to the stock.
 
 For the tissue samples created previously (by bulk and single entry), we will create stocks for library preparation.
 
+## 4.1 Bulk Propagate Samples
+
+For three of the tissues you created in the previous section, create a stock.
+
 1. On the _List Samples_ page, enter your project name in the search box.
+<<<<<<< HEAD
```
Comment (tonydebat) [inline 4-samples.md]: Leftover from a previous merge?

File: 4-samples.md [hunk 2]
```diff
@@ -15,31 +15,86 @@ to the stock.
...
-1. Check the boxes for the stock samples (propagated and received).
+1. Check the boxes for the stock samples (propagated and received). These are
+the samples that end in D_S1:
+  * `PROJ_0001_Br_P_nn_1-1_D_S1`
+  * `PROJ_0001_Ly_R_nn_1-1_D_S1`
+  * `PROJ_0002_Br_P_nn_1-1_D_S1`
+  * `PROJ_0002_Ly_R_nn_1-1_D_S1`
 1. From the _Bulk actions_ dropdown at the bottom, select _Update selected_ and click _Go_.
-1. Change the _QC Passed?_ column to `true` for all rows.
+<<<<<<< HEAD
```
Comment (tonydebat) [inline 4-samples.md]: More previous merge leftovers.

**Comment by tonydebat** [comment]
👍 After removing '<<<< HEAD' strings

**Comment by apmasell** [comment]
:+1: same as Tony with the merge markers

**Pull request merged**